src/view.rs

```diff
@@ -859,3 +859,23 @@ mod view_response_builder_tests {
         assert_eq!(expected, got);
     }
 }
+
+pub struct AllDocumentsViewValue {
```

Comment:
Got it. I was thinking `_all_docs` returns a kind of stub document—probably because my mind is addled from working with attachments a lot lately.

Another point I was wrong about was using `HashMap` because the result is ordered. It should probably be a `Vec`—same as what `UnreducedView` uses. So just as you said, `ViewResponse` could be reused under the hood.